section_faiss_vector_index.go

```diff
@@ -382,15 +419,41 @@ func (v *vectorIndexOpaque) grabBuf(size int) []byte {
 	return buf[0:size]
 }
 
-func getIndexType(nVecs int) (string, bool) {
+// Return the number of centroids for an IVF index.
+func getNumCentroids(nVecs int) int {
```

Comment:
Wonder why this code is showing up here, perhaps this commits needs a rebase now that the base branch has moved from centroids2 to master.